src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/db/repositories/WorkflowRunRepository.kt

```diff
@@ -93,16 +98,30 @@ class OrderlyWebWorkflowRunRepository : WorkflowRunRepository
                 .and(
                     if (namePrefix != null)
                     {
-                        lower(Tables.ORDERLYWEB_WORKFLOW_RUN.NAME).startsWith(namePrefix.toLowerCase())
+                        lower(ORDERLYWEB_WORKFLOW_RUN.NAME).startsWith(namePrefix.toLowerCase())
                     }
                     else
                     {
                         noCondition()
                     }
                 )
-                .orderBy(Tables.ORDERLYWEB_WORKFLOW_RUN.DATE.desc())
+                .orderBy(ORDERLYWEB_WORKFLOW_RUN.DATE.desc())
 
             return result.fetchInto(WorkflowRunSummary::class.java)
         }
     }
+
+    override fun updateWorkflowRun(key: String, status: String)
+    {
+        JooqContext().use {
+            val result = it.dsl.update(ORDERLYWEB_WORKFLOW_RUN)
+                .set(ORDERLYWEB_WORKFLOW_RUN.STATUS, status)
+                .where(ORDERLYWEB_WORKFLOW_RUN.KEY.eq(key))
+                .execute()
+            if (result == 0)
+            {
+                throw UnknownObjectError(key, "workflow")
```

Comment:
I think the text `workflow ` on the throwable should be `update workflow` or `update-workflow` to be more specific.

Reply:
I think that parameter does usually refer to the type of object rather than the action being taken, so probably more consistent to keep it as is.

Reply:
Yep: "workflow" works here because then the [generated error message](https://github.com/vimc/orderly-web/blob/master/src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/errors/UnknownObjectError.kt) will be "Unknown workflow : blah"